backend/app/services/active_user_service.rb

```diff
@@ -17,37 +17,46 @@ def self.active_user(request)
                     request
                 )
             utorid, _password = credentials.split(':')
-            user = User.find_by(utorid: utorid)
+            @user = User.find_by(utorid: utorid)
 
-            # Every user has the TA role
-            roles = user ? user.roles + %w[ta] : %w[ta]
-
-            roles += %w[instructor] if Instructor.find_by(utorid: utorid)
-            # Check if they are listed as an instructor for any course
-            if Rails.application.config.always_admin.include?(utorid)
-                roles += %w[admin]
-            end
-
-            return User.new(utorid: utorid, roles: roles)
+            return User.new(utorid: utorid, roles: grant_roles)
         end
 
         if (Rails.application.config.respond_to? :active_user_override) &&
                Rails.application.config.active_user_override
-            user = User.find_by(id: Rails.application.config.active_user_id)
-            return user if user
-
-            # If we're here, the database doesn't have the specified active user,
-            # so createa fake one with all the permissions. We *do not* save this
-            # user to the database.
-            return(
-                User.new(
+            @user = User.find_by(id: Rails.application.config.active_user_id)
+            if @user
+                return User.new(
+                  utorid: @user.utorid, roles: grant_roles
+              )
+            else
+                # If we're here, the database doesn't have the specified active user,
+                # so createa fake one with all the permissions. We *do not* save this
+                # user to the database.
+                return(
+                  User.new(
                     utorid: 'defaultactive', roles: %w[admin instructor ta]
                 )
-            )
+              )
```

Comment:
Some weird spacing is going on here...did you run Prettier with the Ruby plugin installed?

Reply:
This is auto-corrected by rubocop. The previous indentation did not pass ruby linting tests defined in tests/run-ruby-liniting-tests.sh.

```
snowsr@ADR-LINUX:~/projects/tapp-accel/tests$ ./run-ruby-linting-tests.sh
This script assumes that all docker images are up and running.
If they are not, please run `docker-compose up` and rerun this script.
Running Unit Tests...
WARNING: The RAILS_SERVE_STATIC_FILES variable is not set. Defaulting to a blank string.
WARNING: The RAILS_LOG_TO_STDOUT variable is not set. Defaulting to a blank string.
WARNING: The ALLOW_BASIC_AUTH variable is not set. Defaulting to a blank string.
WARNING: The TAPP_ADMINS variable is not set. Defaulting to a blank string.
Creating tapp_backend_run ... done
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that can also opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

Gemspec/DateAssignment: # (new in 1.10)
  Enabled: true
Style/HashConversion: # (new in 1.10)
  Enabled: true
For more information: https://docs.rubocop.org/rubocop/versioning.html
Inspecting 121 files
.......................................................................C.................................................

Offenses:

app/services/active_user_service.rb:40:13: C: [Correctable] Layout/ClosingParenthesisIndentation: Indent ) to column 14 (not 12)
            )
            ^

121 files inspected, 1 offense detected, 1 offense auto-correctable
```